src/types/layers.ts

```diff
@@ -26,3 +28,11 @@ const layerOrder: Record<Layer, number> = {
 
 export const orderByLayer = (itemA: HasLayer, itemB: HasLayer): number =>
   layerOrder[itemA.layer] - layerOrder[itemB.layer]
+
+const layersWithEncryptedTransactions: Layer[] = [Layer.sapphire]
+
+export const doesLayerSupportEncryptedTransactions = (layer: Layer): boolean =>
+  layersWithEncryptedTransactions.indexOf(layer) !== -1
```

Comment:
optional
`.includes(layer)` is more readable

Reply:
Done